### ISSUE TITLE
Question - Gnome 3.29

### ISSUE THREAD

**Issue by aspann** [hhfeuer/Gentoo-Gnome-3.28-X#7]
Hey,
sorry for opening an issue for that, it's just a qestion..

I've read in another issue in your 3.26 overlay, that you're working on an overlay for 2.29. Is that true? If you need any help on testing or bugfinding, just let me know. (Btw. I fixed my pyblake2 issue..)

And many thanks for adding the evolution stuff.. ;-)

**Comment by hhfeuer** [comment]
No, must have been a typo. 3.29 is a development version, I'm focusing on stable versions, so 3.30 will be my next aim starting in september.

**Comment by aspann** [comment]
great to hear!

If you need any help on testing, etc. just let me know. I stay tuned. :)
